src/EditCocktail/EditCocktail.js

```diff
@@ -92,8 +98,13 @@ const EditCocktail = ({ choosenCocktail, updateCocktail, updateSteps }) => {
   };
 
   const submitEdit = () => {
-    console.log(allIngredients);
-    console.log(steps);
+    if (allIngredients.length === 0 || steps.length === 0) {
```

Comment:
We also need to add the cocktailName and check it!

Reply:
Cool, I will do that now